frontend/amundsen_application/static/js/components/OwnerEditor/index.tsx

```diff
@@ -238,21 +241,19 @@ export class OwnerEditor extends React.Component<
     );
   };
 
-  render() {
-    const { isEditing, readOnly, resourceType } = this.props;
-    const { errorText, itemProps } = this.state;
-    const hasItems = Object.keys(itemProps).length > 0;
-
-    if (errorText) {
-      return (
-        <div className="owner-editor-component">
-          <span className="status-message">{errorText}</span>
-        </div>
-      );
-    }
+  renderOwnersSection = (section: OwnerCategory | null) => {
+    const { resourceType } = this.props;
+    const { itemProps } = this.state;
 
-    const ownerList = hasItems ? (
+    return (
       <ul className="component-list">
```

Comment:
can you add a css class to the `<ul>` here that sets `margin-bottom: $spacer-1` (8px) but then sets it to 0 for the last-child? the larger sections have their own spacing so we want to skip it for the last item